hs_core/templates/search/indexes/hs_core/baseresource_text.txt

```diff
@@ -0,0 +1,38 @@
+{{ object.short_id }}
+{{ object.doi }}
+{{ object.metadata.title }}
+{{ object.first_creator }}
+{{ object.metadata.description }}
+{% for creator in object.metadata.creators.all %}
+{{ creator.name }}
+{% endfor %}
+{% for contributor in object.metadata.contributors.all %}
+{{ contributor.name }}
+{% endfor %}
+{% for subject in object.metadata.subjects.all %}
+{{ subject }}
+{% endfor %}
+{{ object.metadata.publisher.name }}
+{{ object.first_creator.organization }}
+{{ object.first_creator.email }}
+{{ object.public }}
+{{ object.discoverable }}
+{{ object.created }}
+{{ object.modified }}
+{{ object.rating_sum }}
+{% for coverage in object.metadata.coverages.all %}
+{{ coverage.value }}
+{% endfor %}
+{% for format in object.metadata.formats.all %}
+{{ format.value }}
+{% endfor %}
+{% for identifier in object.metadata.identifiers.all %}
+{{ identifier.name }}
+{% endfor %}
+{{ object.metadata.language.code }}
+{% for source in object.metadata.sources.all %}
+{{ source.derived_from }}
+{% endfor %}
+{% for relation in object.metadata.relations.all %}
+{{ relation.value }}
+{% endfor %}
```

Comment:
@xuezhaokun Can you add formatting/proper indentation into this template as @ironfroggy suggested above? I am going through all comments to check whether they have been addressed.